source/includes/Orders/order.md

```diff
@@ -0,0 +1,116 @@
+# Order
+
+## Overview
+
+An **Order** is an action that a provider requests be taken regarding a patient.
+
+For example a provider could order a Complete Blood Count laboratory panel for a patient.
+
+An Order only records an intention, not whether or not the action is carried out. The results of an Order are typically recorded later as Observations.
+
+## Available operations.
+
+1. [List orders](#list-orders)
+2. [Create an order](#create-an-order)
+3. [Update an order](#update-an-order)
+4. [Delete an order](#delete-an-order)
+
+### List orders
+
+* Fetch all non-retired roles that match any specified parameters otherwise fetch all non-retired roles. 
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of order
+
+    ```console
+    GET /order?q=penicillin
+     ```
+
+* #### List a particular order
+
+    Retrieve a particular order.
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    ```console
+    GET /order/:target_order_uuid
+    ```
+
+### Create an order
+
+* To create a role you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *encounter* | `Encounter UUID` | the encounter for this order
+    *orderType* | `String` | the type of the order
+    *action* | `String` | the action for the order
+    *accessionNumber* | `String` | 
+    *patient* | `Patient UUID` | the patient for the order
+    *concept* | `Concept UUID` | the concept of the order
```

Comment:
UUID for the concept of the order; the item being requested (e.g., "serum creatinine")